odxtools/cli/_print_utils.py

```diff
@@ -228,9 +228,10 @@ def extract_parameter_tabulation_data(parameters: List[Parameter]) -> Table:
             value_type.append(None)
             dop.append(None)
 
+    for lst in [byte, semantic, bit_length, value, value_type, data_type, dop]:
+        lst[:] = ["" if x is None else x for x in lst]
```

Comment:
great that you made this code here more readably. line 232 should probably better be
```suggestion
        # replace `None` entries by empty strings
        lst = ["" if x is None else x for x in lst]
```

Reply:
I have tried using simple `lst` it doesn't override the list in-place.

Reply:
why does it need to be overwritten in-place? (memory wise it should not make any difference because the current solution creates a temporary list as well and also the required memory hopefully is miniscule...)

Reply:
Yes it does create a temporary list in both case, but in case of `list[:]` it copies the temporary list to the original address. So in a way I am able to edit the list in it's place.

Reply:
okay, I had hands-on look and I now understand why the in-place copy of the list is necessary. That said, declaring `List[Any]` for any of the variables above just causes `mypy` to give up here, so -- while not optimal either -- I prefer to use `List[Optional[int]]` above and add a `# type: ignore[attr-defined, index]` comment here...

Reply:
okay thanks for the that. I am new to this static type checks. I'll push the changes